server/src/server.js

```diff
@@ -175,7 +175,7 @@ app.get("/", function (req, res) {
 app.get("/:id/:domain", function (req, res) {
   let shotId = req.params.id + "/" + req.params.domain;
   Shot.get(req.backend, shotId).then((shot) => {
-    if (! shot) {
+    if (! shot || Object.getOwnPropertyNames(shot._clips).length === 0) {
```

Comment:
No, it's a real instance (created by `Shot.get()`).  Also if it was JSON then it would be `shot.clips` and not `shot._clips`